server/config/routes.js

```diff
@@ -254,8 +254,10 @@ exports.configure = function configure(app) {
   app.post('/posting_donation/', donations.post);
 
   /*  Inventory and Stock Managment */
+  app.post('/inventory/metadata', inventory.createInventoryItems);
```

Comment:
The initial inventory API was written before we had API standards.  Why do you think about naming this API `/inventory/` instead of `/inventory/metadata/`?